app/models/transaction.rb

```diff
@@ -11,10 +11,12 @@ def amount=(value)
   private
 
   def return_amount_with_two_decimal(value)
-    if value.is_a?(String)
-      errors[:amount]
+    if value.to_s.include?('.')
+      num = value.to_s.split('.')
+      num[1] = num[1][0..1]
```

Comment:
Logic here is to remove all the extra numbers and leave only two numbers after point;
Also, remove auto rounding, so when we trying to pass ```2.129``` for example, we will get ```2.12```, instead of ```2.13```